runtime/local/src/lib.rs

```diff
@@ -845,52 +848,70 @@ impl Default for ProxyType {
                     RuntimeCall::System(..)
-                        | RuntimeCall::Utility(..)
                         | RuntimeCall::Timestamp(..)
+                        | RuntimeCall::Scheduler(..)
+                        | RuntimeCall::Proxy(..)
                         | RuntimeCall::Grandpa(..)
                         // Skip entire Balances pallet
                         | RuntimeCall::Vesting(pallet_vesting::Call::vest{..})
-                        | RuntimeCall::Vesting(pallet_vesting::Call::vest_other{..})
-                        // Specifically omitting Vesting `vested_transfer`, and `force_vested_transfer`
+				        | RuntimeCall::Vesting(pallet_vesting::Call::vest_other{..})
+				        // Specifically omitting Vesting `vested_transfer`, and `force_vested_transfer`
                         | RuntimeCall::DappsStaking(..)
                         // Skip entire EVM pallet
                         // Skip entire Ethereum pallet
                         // Skip entire EthCall pallet
                         | RuntimeCall::BaseFee(..)
                         // Skip entire Contracts pallet
-                        // Skip entire Assets pallet
-                        | RuntimeCall::Scheduler(..)
                         | RuntimeCall::Democracy(..)
                         | RuntimeCall::Council(..)
                         | RuntimeCall::TechnicalCommittee(..)
                         | RuntimeCall::Treasury(..)
-                        | RuntimeCall::Proxy(..)
                         | RuntimeCall::Xvm(..)
                 )
             }
+            // All Runtime calls from Pallet Balances allowed for proxy account
+            ProxyType::Balances => {
+                matches!(c, RuntimeCall::Balances(..))
+            }
+            // All Runtime calls from Pallet Assets allowed for proxy account
+            ProxyType::Assets => {
+                matches!(c, RuntimeCall::Assets(..))
+            }
             ProxyType::Governance => {
                 matches!(
                     c,
                     RuntimeCall::Democracy(..)
                         | RuntimeCall::Council(..)
                         | RuntimeCall::TechnicalCommittee(..)
                         | RuntimeCall::Treasury(..)
-                        | RuntimeCall::Utility(..)
                 )
             }
+            // Only reject_announcement call from pallet proxy allowed for proxy account
             ProxyType::CancelProxy => {
                 matches!(
                     c,
                     RuntimeCall::Proxy(pallet_proxy::Call::reject_announcement { .. })
                 )
             }
+            // All runtime calls from pallet DappStaking allowed for proxy account
             ProxyType::DappsStaking => {
-                matches!(c, RuntimeCall::DappsStaking(..) | RuntimeCall::Utility(..))
+                matches!(c, RuntimeCall::DappsStaking(..))
+            }
+            ProxyType::StakerRewardClaim => {
```

Comment:
Let's be consistent with the comments.
Either all should have them or none.

Also for the enum values - e.g. in Shiden we have enum comments, but in Shibuya and local we don't.